crates/sock-prot/src/protector/apple.rs

```diff
@@ -67,15 +66,15 @@ use objc_foundation::{INSString, NSString};
 #[cfg(any(target_os = "ios", target_os = "tvos"))]
 use version_compare::{compare_to, Cmp};
 
-pub(crate) fn bind_to_tun(sock: NativeSocket, tunnel_interface: u64) -> io::Result<()> {
-    telio_log_debug!(
+fn bind_to_tun(sock: NativeSocket, tunnel_interface: u64) -> io::Result<()> {
+    log::debug!(
```

Comment:
I'm not sure if this will hook into the telio logging facilities and send the log messages to apps callback. I think you will have to switch this to analogous macros from the `tracing` crate.

Reply:
I tried to avoid that, `tracing` is a larger thing and quite tied to tokio, which ideally shouldn't be the dependency in the first place (tokio I mean). Tracing is used by telio but not by the other lib.
There's a way for `log` macros to work within telio, by the compatibility layer https://docs.rs/tracing-log/latest/tracing_log/#convert-log-records-to-tracing-events. Do you think we should use that?

Reply:
`tracing_log` already [depends](https://github.com/tokio-rs/tracing/blob/master/tracing-log/Cargo.toml#L26) on `tracing_core` so imo you can just as well depend on `tracing`. Unless we know about some very likely future consumer of this crate that would find this to be a big issue.

Reply:
I mean using `tracing_log` from telio, for capturing the logs. This will save `sock-prot` from using tracing.
As I said we already have a lib that needs to use it and is not using tracing